sample-apps/segmentation_spleen_tta/validate_TTA.py

```diff
@@ -0,0 +1,92 @@
+import argparse
```

Comment:
lets think about where to put these test utils.. for now let it be part of app folder